site/docs/stylesheets/extra.css

```diff
@@ -21,3 +21,31 @@ img.bordered {
 .md-typeset td code {
   word-break: revert;
 }
+
+.md-typeset .tabbed-labels>label {
+  background-color: #efefef;
+  margin-right: 2px;
+}
+
+.md-typeset .tabbed-set>input:first-child:checked~.tabbed-labels>:first-child,
```

Comment:
This is a bit disappointing, but there is no CSS class specific to selected tabs, so basically the stylesheet attempts to apply the style to the first 20 tabs of a tabset, but only if they are selected.

Reply:
Oh - I see, it's because of the two distinct sets of elements.

Mind adding a comment that there are two sets of elements? It took me a while to realize that.

Reply:
What do you mean by two sets? I count 20 rules. Are you referring to the next-sibling combinator `~`?

Reply:
The sets of `input` and `label` elements